RELEASE-NOTES.md

```diff
@@ -1,6 +1,12 @@
 ## RELEASE NOTES
+
+### Version 2.70.1-alert-service-update
```

Comment:
As a minimum, this is a minor release, not a patch. It should go up to `2.71.0`.

Reply:
Also, it doesn't look like this is to do with the alert service...?

Reply:
Agree with first comment. Regarding second comment, even though the changes are within workallocation service, the changes have been made to the alert service and its ability to display messages in certain circumstances. Plus it's a reference to the earlier ccd-ui-toolkit change made

Reply:
I don't see any changes in this release to the alert service. I see it being _used_ by the WA stuff, but no changes to the alert service itself...?

Besides, the name should describe what the primary purpose of this release is for and that looks to be task event completion - the PR is even named for that JIRA. :)

Reply:
Scrub that, this includes a breaking change. It should be version `3.0.0`.